.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Motivation and Context
+
+<!--
+  Why is this change required? What problem does it solve?
+  This can be omitted if a linked issue is provided 
+-->
+
+## Description
+
+<!-- Describe your changes -->
+
+## Test Plan
+
+<!-- Please describe in detail how you tested your changes. -->
+
+## Screenshots (if appropriate):
+
+<!-- 
+  This allows reviewers to begin reviewing your work 
+  without checking out your branch locally
+-->
+
+## Related Issue
```

Comment:
I think just the Github auto linking to the related issue should be enough rather than an h2 section for it.

Reply:
I don't see the auto linking working in a lot of PRs, as a reviewer, I'd like to just click on the link to the related issue. It seems to be a recurring preference, for example in the most recent PR:

<img width="941" alt="Screen Shot 2022-05-24 at 8 33 23 am" src="https://user-images.githubusercontent.com/78794805/169915394-7d20b58a-5ec2-4cf8-b541-89e261a84679.png">